app/src/main/java/org/oppia/app/splash/SplashActivity.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.app.splash
+
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.FragmentTransaction
+
+import android.os.Bundle
+import android.view.WindowManager
+import com.example.myapplication.SplashFragment
+import org.oppia.app.R
+
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash)
+
```

Comment:
Ditto here. For consistency, let's use one space between blocks of code when separating things out.

Ditto elsewhere in this PR.

Reply:
removed extra spaces all over